.github/workflows/publish.yml

```
@@ -1,7 +1,7 @@
 name: Publish
 on:
   push:
-    branches: master
+    branches: 2-x-stable
```

This is wrong, sorry. 
```suggestion
    branches: master
```